### ISSUE TITLE
No series file found

### ISSUE THREAD

**Issue by Rudloff** [MusicPlayerDaemon/MPD#474]
Hello,

I'm trying to build v0.21.4 for F-Droid (https://github.com/MusicPlayerDaemon/MPD/issues/299) and `build.py` fails with this error:
```
No series file found
Traceback (most recent call last):
  File "build.py", line 181, in <module>
    x.build(toolchain)
  File "/home/pierre/www/MPD/python/build/autotools.py", line 59, in build
    build = self.configure(toolchain)
  File "/home/pierre/www/MPD/python/build/autotools.py", line 24, in configure
    src = self.unpack(toolchain)
  File "/home/pierre/www/MPD/python/build/project.py", line 61, in unpack
    push_all(toolchain, path, self.patches)
  File "/home/pierre/www/MPD/python/build/quilt.py", line 9, in push_all
    run_quilt(toolchain, src_path, patches_path, 'push', '-a')
  File "/home/pierre/www/MPD/python/build/quilt.py", line 6, in run_quilt
    subprocess.check_call(['quilt'] + list(args), cwd=cwd, env=env)
  File "/usr/lib/python3.5/subprocess.py", line 271, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['quilt', 'push', '-a']' returned non-zero exit status 2
```
I have quilt 0.65 installed.

**Comment by MaxKellermann** [comment]
How to reproduce?

**Comment by Rudloff** [comment]
I am running this on Debian 9.7:
```
../../android/build.py /home/pierre/www/android-sdk/ /home/pierre/www/android-sdk/ndk-bundle/ armeabi-v7a
```
The error happens just after `download http://curl.haxx.se/download/curl-7.62.0.tar.xz`.

**Comment by MaxKellermann** [comment]
I tried back and forth, but I can't reproduce this. The `series` file does exist in `src/lib/curl/patches/`.
What you can do is edit `python/build/project.py` and dump the value of the variables `srcdir` and `patches`. Or maybe insert `strace` into `python/build/quilt.py` and see where Quilt looks for its `series` file.

**Comment by Rudloff** [comment]
Here are the values:
* `srcdir`: `/home/pierre/www/MPD`
* `patches`: `/home/pierre/www/MPD/src/lib/curl/patches`

I also tried running `quilt --trace`:  https://gist.github.com/Rudloff/3d311cf7d87e8842a421a56a332f39e3
And I finally found the problem! I had a `.quiltrc` file containing `QUILT_PATCHES=debian/patches` so quilt was looking in the wrong folder.
I don't remember creating this file but it was probably created automatically by Debian tools like debuild.

Thanks for your help and sorry for not thinking of this earlier.
The app now builds correctly :slightly_smiling_face:
